internal/gl/gl_webgl_wasm.go

```diff
@@ -1133,16 +1131,16 @@ func Uniform1fv(dst Uniform, src []float32) {
 // 	c.Call("uniform3f", dst.Value, v0, v1, v2)
 // }
 
-// func Uniform3fv(dst Uniform, src []float32) {
-// 	// c.Call("uniform3fv", dst.Value, src)
+func Uniform3fv(dst Uniform, src []float32) {
+	// c.Call("uniform3fv", dst.Value, src)
 
-// 	SliceToTypedArray(src)
-// 	c.Call("uniform3fv", dst.Value, jsMemoryBufferVec3)
+	// SliceToTypedArray(src)
+	// c.Call("uniform3fv", dst.Value, jsMemoryBufferVec3)
 
-// 	// array, length := SliceToTypedArray(src)
-// 	// subarray := array.Call("subarray", 0, length)
-// 	// c.Call("uniform3fv", dst.Value, subarray)
-// }
+	array, length := SliceToTypedArray(src)
+	subarray := array.Call("subarray", 0, length)
+	c.Call("uniform3fv", dst.Value, subarray)
```

Comment:
Didn't really know which portion to uncomment here but this seemed to work.

Reply:
That will work. The reason I commented them out was so that I could convert them to a faster syntax, if/when I ever re-enabled them. If you look at: https://github.com/unitoftime/glitch/pull/15/files#diff-998ed2d5fa43131e5d8b47cd90ef6527cc6c29951cc5878b5280a58f2f8666e8L1059

You'll see that it uses syntax `fnTexSubImage2D.Invoke(...)` which uses cached data to call the webgl function (and thus will create less garbage). The `Call("", ...)` syntax does effectively the same thing but has to rebuild some of the js args every time (specifically the string, which requires a copy to the JS world).

Either:
1. You can convert it to the same syntax of `fnTexSubImage2D` by creating some other globally bound variable, maybe: `fnUnifirom3fv` or something then doing the invoke on it.
2. You can just leave it as is and I can do it after merged (wont take long)

Reply:
Addressed in https://github.com/unitoftime/glitch/pull/15/commits/ad5d1d7e13b2f46228d054082693abc15968dd41 - again, this is mostly over my head so I just went with what appears to work, LMK if you think it needs adjusting.